packages/transition-backend/src/services/osm/__tests__/OsmGetStreetsAroundPoint.test.ts

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2024, Polytechnique Montreal and contributors
+ *
+ * This file is licensed under the MIT License.
+ * License text available at https://opensource.org/licenses/MIT
+ */
+import { getStreetsAroundPoint } from '../OsmGetStreetsAroundPoint';
+import OsmDownloader from 'chaire-lib-common/lib/utils/osm/OsmOverpassDownloader';
+import * as Status from 'chaire-lib-common/lib/utils/Status';
+
+jest.mock('chaire-lib-common/lib/utils/osm/OsmOverpassDownloader', () => {
+    return { downloadGeojson: jest.fn() };
+});
+
+describe('getStreetsAroundPoint', () => {
+    const mockedDownloadGeojson = OsmDownloader.downloadGeojson as jest.MockedFunction<typeof OsmDownloader.downloadGeojson>;
+    it('should return mocked streets around a given point', async () => {
+        mockedDownloadGeojson.mockResolvedValueOnce({
```

Comment:
Maybe a test with a valid, but unexpected geojson (like no street name)

Reply:
this service does not check for street name, only for streets around node. The check for name is in the  NodeEditForm, which is not yet tested...